cfgov/unprocessed/apps/mmt-cost-of-credit-tool/webpack-config.js

```diff
@@ -0,0 +1,215 @@
+const APP_NAME = 'mmt-cost-of-credit-tool';
+const PUBLIC_PATH = `/static/apps/${APP_NAME}/js/`;
+const { paths } = require('../../../../config/environment');
+const { LAST_2_IE_11_UP } = require('../../../../config/browser-list-config');
+const webpack = require('webpack');
+const path = require('path');
+const TerserPlugin = require('terser-webpack-plugin');
+const CopyPlugin = require('copy-webpack-plugin');
+const WorkboxPlugin = require('workbox-webpack-plugin');
+const { InjectManifest } = WorkboxPlugin;
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const OptimizeCssAssetsPlugin = require('optimize-css-assets-webpack-plugin');
+
+// const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
+
+// Used for toggling debug output. Inherit Django debug value to cut down on redundant environment variables:
+const { DJANGO_DEBUG: DEBUG = false, NODE_ENV = 'development', ANALYZE = false } = process.env;
+
+const COMMON_BUNDLE_NAME = 'common.js';
+//const SERVICE_WORKER_DESTINATION = '../../../../jinja2/v1/mmt-cost-of-credit-tool/service-worker.js';
+const SERVICE_WORKER_DESTINATION = 'service-worker.js';
+
+const AUTOLOAD_REACT = new webpack.ProvidePlugin({
+  React: 'react',
+});
+
+const ENVIRONMENT_VARIABLES = new webpack.DefinePlugin({
+  'process.env.NODE_ENV': JSON.stringify(NODE_ENV),
+  'process.env.DEBUG': JSON.stringify(DEBUG),
+});
+
+const COPY_PWA_MANIFEST = new CopyPlugin([
+  {
+    from: path.join(__dirname, 'manifest.json'),
+    to: '..',
+  },
+]);
+
+const EXTRACT_CSS = new MiniCssExtractPlugin({
+  moduleFilename: ({ name }) => `${name.replace('js', 'css')}`,
+});
+
+const GENERATE_SERVICE_WORKER = new InjectManifest({
+  swSrc: 'cfgov/unprocessed/apps/mmt-cost-of-credit-tool/js/sw.js',
+  swDest: SERVICE_WORKER_DESTINATION,
+});
```

Comment:
This can be removed or commented out unless you're sure the app needs to work offline. Otherwise a service worker is just going to make pushing updates harder due to the aggressive local caching we've got it set to do.